schemas/oic.r.account-request.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema" : "http://json-schema.org/draft-04/schema#",
+  "description" : "Copyright (c) 2017 Open Connectivity Foundation, Inc. All rights reserved.",
```

Comment:
Should all of the copyrights but updated to 2018?

Reply:
Done.